pages/_formats/genome-interval-coordinates.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "GA4GH Intervals"
+layout: default
+date: 2019-01-23
+author: "@mbaudis"
+excerpt_separator: <!--more-->
+category:
+  - formats
+tags:
+  - coordinates
+---
+
+## {{ page.title }}
+
+#### Status: __draft__
+
+#### Contributors
+
+* Andy Yates
+
+#### Definition
+
+Two integers that define the start and end positions of a range of residues, possibly with length zero, and specified using interbase coordinates. Coordinates are assumed to be positioned on a non-circular sequence.
+
+#### Model
+
+* start (uint64) start position >= 0 (required)
+* end (uint64) end position >= start (required)
+
+#### Background
+
+When humans refer to a range of residues within a sequence, the most common convention is to use an interval of ordinal residue positions in the sequence i.e. start counting residues from 1. This system is also referred to as "1-start, fully-closed", biological coordinates and "Ensembl style". While natural for humans, this convention has several shortcomings for data modelling and programming. GA4GH prefers the use of interbase or "0-based, half-open" coordinates (also known as Chado or "UCSC style") and strongly advises that all future products prefer their use for future products unless the product visually displays data to a human. Interbase coordinates refer to the zero-width points before and after residues. An interval of interbase coordinates permits referring to any span, including an empty span, before, within, or after a sequence.
+
+#### The Interbase Coordinate System
+
+While interbase is numerically equivalent to "0-start, fully-closed" they are semantically different. Interbase does not refer to residues and therefore can model events occurring between residues, the start and end of a sequence. For non-circular sequences the following holds true.
```

Comment:
What it's numerically equivalent to is 0-start **half-open**.

Reply:
Fixed. Also, 2 pages now; one more general/recommendation; one (@jmarshall 's) with more details to interbase use etc.

IMO this page here could be condensed a bit now.